lib/beaker/host/windows.rb

```diff
@@ -39,9 +39,10 @@ def self.foss_defaults
         'user'              => 'Administrator',
         'group'             => 'Administrators',
         'puppetpath'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetconfdir'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetcodedir'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
+        'puppetconfdir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
+        'puppetcodedir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
         'hieraconf'         => '`cygpath -smF 35`/Puppetlabs/puppet/etc/hiera.yaml',
+        'hieradatadir'      => '`cygpath -smF 35`/PuppetLabs/puppet/etc/hieradata',
```

Comment:
he said that while it's not necessary, if/when we start testing hiera on Windows, Beaker will need this setting to find the default config file.  I thought it would be better to take on any maintenance burden right now for that, and try to future-proof ourselves a little bit with this ride-along change.  @joshcooper am I getting your message wrong?